### PR TITLE
Annotations - Hive

### PR DESCRIPTION
**Pull request by wild-endeavor**
https://github.com/lyft/flyte/issues/631

### CODE REVIEW

File: tests/flytekit/unit/annotated/test_type_hints.py [hunk 1]
```diff
@@ -273,13 +273,13 @@ def t1() -> datetime.datetime:
...
-    def my_wf() -> str:
+    def my_wf() -> FlyteSchema:
         dt = t1()
         return sql(ds=dt)
 
     with task_mock(sql) as mock:
-        mock.return_value = "Hello"
-        assert my_wf() == "Hello"
+        mock.return_value = pandas.DataFrame(data={"x": [1, 2], "y": ["3", "4"]})
+        assert (my_wf().open().all() == pandas.DataFrame(data={"x": [1, 2], "y": ["3", "4"]})).all().all()
```
Comment (wild-endeavor) [inline tests/flytekit/unit/annotated/test_type_hints.py:282]: is this the right way to use the FlyteSchema?

File: flytekit/taskplugins/hive/task.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        DROP TABLE {{{{ .PerRetryUniqueKey }}}};
+        """
+
+    def __init__(
+        self,
+        name: str,
+        cluster_label: str,
+        inputs: Dict[str, Type],
+        select_query: str,
+        output_schema_type: Type[FlyteSchema],
```
Comment (kumare3) [inline flytekit/taskplugins/hive/task.py:88]: What if we dont want an output? I guess that will be a separate type of hiveTask?

File: tests/flytekit/unit/annotated/test_type_hints.py (hunk 1 above)
Comment (kumare3) [reply tests/flytekit/unit/annotated/test_type_hints.py:283]: yes, but you cannot just compare 2 pandas dataframes like this
you have to compare, `==` which returns a new dataframe, that you assert has all `True`

Look at some examples of how I am doing it

**Comment by kumare3** [comment]
cool 🆒

File: flytekit/taskplugins/hive/task.py (hunk 2 above)
Comment (wild-endeavor) [reply flytekit/taskplugins/hive/task.py:88]: This is the select task though... if they don't want an output, they can always use the barebones hive task.

**Pull request merged**